src/templates/queues/list.html

```diff
@@ -0,0 +1,7 @@
+{% extends "queues/base_queues.html" %}
```

Comment:
this should just extend base. Don't think we need the intermediary template

Reply:
Pulled `queues/base_queues.html` and set to extend `base.html`